docs/components/DataList/DataList.stories.mdx

```diff
@@ -304,3 +304,53 @@ want to inform the user about something related to the Datalist.
   </DataList.StatusBar>
 </DataList>
 ```
+
+### Sorting
+
+The `sorting` prop is an object that configures the sorting functionality of the
+DataList.
+
+The `state` property is an object that holds the current sorting state of the
+DataList. The `onSort` property is a function that is called when the user
+clicks on a sortable column. Lastly, `sortable` is an array that holds
+information about which columns are sortable.
+
+Each column is identified by a `key` and will have a `sortType` property that
```

Comment:
One comment that can be handled by docs is that `toggle` doesn't update the arrow / work for all options when the toggle sort has more than 2 options

Reply:
ex. changing the story to have `label` as a `toggle` instead of dropdown

Reply:
Hmm... I wonder if that is something we should be putting in the docs? I feel like we could/should fix that in the future and I can write a ticket for it. The component is also still technically in Alpha so I hesitate to point out behaviour that may change later on.
How strongly do you feel about including this @MichaelParadis ?

Reply:
I am good with just approving. I have a slight concern about people having problems due to misuse but I think it is extremely unlikely (toggles are to switch between 2 states so why use it for more than 2)

Reply:
Exactly - ok I agree. Thanks Michael!